src/rootcheck/unix-process.c

```diff
@@ -19,13 +19,20 @@ static char *_os_get_runps(const char *ps, int mpid)
     char *tmp_str, *nbuf;
     char buf[OS_SIZE_2048 + 1];
     char command[OS_SIZE_1024 + 1];
+    int size = 0;
     FILE *fp;
 
     buf[0] = '\0';
     command[0] = '\0';
     command[OS_SIZE_1024] = '\0';
 
-    snprintf(command, OS_SIZE_1024, "%s -p %d 2> /dev/null", ps, mpid);
+    size = snprintf(command, sizeof(command), "%s -p %d 2> /dev/null", ps, mpid);
```

Comment:
This might be changed to 
```suggestion
    const int size = snprintf(command, sizeof(command), "%s -p %d 2> /dev/null", ps, mpid);
```